src/styles/blocks/starReviewsPrice.scss

```diff
@@ -0,0 +1,77 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  width: 166px;
+  margin-block: 8px;
+  gap: 17px;
+
+  &-amount {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-end;
+    height: 16px;
+    width: 53px;
+    margin: 6px 0 10px;
```

Comment:
Avoid using shorthand properties for margin/padding when only one or two values are needed, as it can lead to confusion and less maintainable code. In this case, you should use `margin-top` and `margin-bottom` separately.